docs/tutorials/gluon/logistic_regression_explained.md

```diff
@@ -0,0 +1,215 @@
+
+# Logistic regression using Gluon API explained
```

Comment:
There is a lot of similarity with the tutorial you have mentioned. My personal pet-peeve with the tutorial in The Straight Dope is that it doesn't use library functions for sigmoid, loss and accuracy. I believe it creates an impression that you need to write a lot of boiler plate code instead of using what MXNet offers. 

Also, there were confusions around number of neurons in forum and stackoverflow which is directly relates to the usage of `Accuracy`, but needs a bit more context to cover.